Src/Plugins/Tests.Security/Validators/SEC101_001.HttpAuthorizationRequestHeaderValidatorTests.cs

```diff
@@ -26,25 +26,6 @@ public class HttpAuthorizationRequestHeaderValidatorTests
         private const string TestKey = "somekey";
         private const string TestHost = "www.host.com";
         private const string TestResource = "/some-path";
-        private const ValidationState ExpectedValidationState = ValidationState.NoMatch;
-
-        [Fact]
-        public void HttpAuthorizationRequestHeaderValidator_Test()
```

Comment:
>HttpAuthorizationRequestHeaderValidator_Test [](http://example.com/codeflow?start=20&length=44)

This test was trying to connect at www.host.com and verifying if that validates with the key `somekey`. Since this isn't bringing any value, i'm removing it.

Reply:
If we still don't undertsand the root cause, should we be deleting this entirely at this point? Or disable to attempt to unblock the pipeline?

Reply:
since we were testing against a real site, it is expected to change from time to time. We don't need a test that we dont know for sure what is the expected result.

it's why i deleted it.

Reply:
It's a good idea in general not to reach across the network with key unit tests, so I think this is probably a good idea. One hint, if you *do* need a host to test against, example.com can be useful. It is intended for allocating (non-functional) URLs, for illustrative purposes (i.e., to serve as examples).